src/components/language-indicator/index.tsx

```diff
@@ -2,14 +2,27 @@ import React, { FC, memo } from 'react';
 import { compose } from 'redux';
 
 import SC from './styled';
-import { TextLanguage } from '../../types';
+import type { TextLanguage } from '../../types';
 
 interface Props {
   textLanguage?: Partial<TextLanguage>;
   selectedLanguage?: string;
   whiteBackground?: boolean;
 }
 
+const languageTitle = (language: string) => {
+  switch (language) {
+    case 'en':
+      return 'English';
+    case 'nb':
+      return 'Norsk bokmål';
+    case 'nn':
+      return 'Nynorsk';
+    default:
+      return '';
```

Comment:
Burde vi velge noen av språkene som default her, istedet for tom streng?

Reply:
Det blir feil tror jeg

Reply:
Dette er kun en forklaring for nb, nn og en